mico-core/src/main/resources/application-dev.properties

```diff
@@ -36,7 +36,7 @@ kubernetes.build-bot.docker-image-repository-url=docker.io/ustmico
 kubernetes.build-bot.docker-registry-service-account-name=build-bot-dockerhub
 kubernetes.build-bot.kaniko-executor-image-url=gcr.io/kaniko-project/executor
 kubernetes.build-bot.build-timeout=600
-kubernetes.build-bot.build-clean-up-by-undeploy=true
+kubernetes.build-bot.build-clean-up-by-undeploy=false
```

Comment:
Should this be merged into mmaster?

Reply:
This was not intended, but I still think it makes sense that the build resources are not deleted immediately in the testing namespace after an undeployment of an application.